.github/workflows/_tauri.yml

```diff
@@ -100,12 +100,12 @@ jobs:
           path: |
             ${{ matrix.pkg-artifact }}
           if-no-files-found: error
-      # Only for builds on main
       - name: Upload Release Assets
-        if: ${{ inputs.release_tag || env.VERSION }}
+        # Only upload the windows build to the drafted release on main builds
+        if: ${{ github.ref_name == 'main' }}
```

Comment:
Slight fix to #4251 -- we probably don't want to overwrite the drafted release MSI on PRs.